lib/block-supports/layout.php

```diff
@@ -280,6 +280,37 @@ function gutenberg_get_layout_style( $selector, $layout, $has_block_gap_support
 				);
 			}
 		}
+	} elseif ( 'grid' === $layout_type ) {
+			$minimum_column_width = ! empty( $layout['minimumColumnWidth'] ) ? $layout['minimumColumnWidth'] : '12rem';
+
+			$layout_styles[] = array(
+				'selector'     => $selector,
+				'declarations' => array( 'grid-template-columns' => 'repeat(auto-fill, minmax(min(' . $minimum_column_width . ', 100%), 1fr))' ),
+			);
```

Comment:
Just noticed that PHPCS is complaining about indentation in this file. While the warnings are about the lines below, it looks like the culprit is this set of lines being one level too far indented 🙂

Reply:
yeah, I just realised my auto-formatter isn't working 😅